### PR TITLE
New Module: na_ontap_quotas

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
- New Module Na_ontap_quotas. Allow for the creating/modifying and deleting of Quotas on ONTAP
- Unit test for Modules

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
- na_ontap_quotas

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by thedoubl3j** [comment]
CI restarted per @gundalow

**Review by thedoubl3j — APPROVED** [review]
module looks good, CRUD followed, tests included look good. lgtm ping @gundalow 

cc @thaumos

**Review by thedoubl3j — COMMENTED** [review]
@carchi8py any particular reason the test for ontap_cluster was removed?

**Comment by carchi8py** [comment]
So looks like in my devel i deleted it, that was on accident, Problem with that is i use the same workspace to push out every change in the last month, so all my pull request now have a delete for this file.

Since i don't want to redo dozens of pull request i'll add a new one in a little bit to re-add the unit test.

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
rebuild_merge

**Comment by carchi8py** [comment]
Just realized i can revert the delete without causing any issues. So i'm going through all the existing pull request and reverting the delete of that one file.

**Review by lonico — APPROVED** [review]
shipit

**Review by lonico — APPROVED** [review]
shipit

**Comment by thedoubl3j** [comment]
cc @thaumos

File: lib/ansible/modules/storage/netapp/na_ontap_quotas.py [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+DOCUMENTATION = '''
+module: na_ontap_quotas
+short_description: NetApp ONTAP Quotas
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.8'
+author: NetApp Ansible Team (@carchi8py) <ng-ansibleteam@netapp.com>
+description:
+- Set/Modify/Delete quota on ONTAP
+options:
```
Comment (gundalow) [inline lib/ansible/modules/storage/netapp/na_ontap_quotas.py:24]: Can you please add `type:` for all new module options.
We are using this to improve the module documentation, thanks.

Comment (carchi8py) [reply lib/ansible/modules/storage/netapp/na_ontap_quotas.py:24]: @gundalow for options with choices should type be str?

Comment (carchi8py) [reply lib/ansible/modules/storage/netapp/na_ontap_quotas.py:24]: I've added it as a str.

**Pull request merged**